packages/suite/src/support/messages.ts

```diff
@@ -9649,6 +9645,18 @@ export default defineMessages({
         id: 'TR_STAKE_CLAIM',
         defaultMessage: 'Claim',
     },
+    TR_STAKE_STAKE_NETWORK: {
+        id: 'TR_STAKE_STAKE_NETWORK',
+        defaultMessage: 'Stake {symbol}',
+    },
+    TR_STAKE_UNSTAKE_NETWORK: {
+        id: 'TR_STAKE_UNSTAKE_NETWORK',
+        defaultMessage: 'Unstake {symbol}',
+    },
+    TR_STAKE_CLAIM_NETWORK: {
```

Comment:
```suggestion
    TR_STAKE_CLAIM_TOKEN: {
```

we will be introducing stablecoin staking, so i think it makes more sense to have token there. ETH and SOL is also token (native)

Reply:
Done